src/System.ServiceModel.Syndication/src/System/ServiceModel/Syndication/SyndicationItemFormatter.cs

```diff
@@ -2,72 +2,40 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Xml;
+using System.Runtime.Serialization;
+using System.Globalization;
+
 namespace System.ServiceModel.Syndication
 {
-    using System;
-    using System.Collections.Generic;
-    using System.Xml;
-    using System.Runtime.Serialization;
-    using System.Globalization;
-    using System.Xml.Serialization;
-    using System.Xml.Schema;
-    using System.Diagnostics.CodeAnalysis;
-    using DiagnosticUtility = System.ServiceModel.DiagnosticUtility;
-    using System.Runtime.CompilerServices;
-
     [DataContract]
     public abstract class SyndicationItemFormatter
     {
-        private SyndicationItem _item;
-
-        protected SyndicationItemFormatter()
-        {
-            _item = null;
-        }
+        protected SyndicationItemFormatter() => Item = null;
 
         protected SyndicationItemFormatter(SyndicationItem itemToWrite)
         {
-            if (itemToWrite == null)
-            {
-                throw DiagnosticUtility.ExceptionUtility.ThrowHelperArgumentNull("itemToWrite");
-            }
-            _item = itemToWrite;
+            Item = itemToWrite ?? throw new ArgumentNullException(nameof(itemToWrite));
         }
 
-        public SyndicationItem Item
-        {
-            get
-            {
-                return _item;
-            }
-        }
+        public SyndicationItem Item { get; private set; }
 
-        public abstract String Version
-        { get; }
+        public abstract string Version { get; }
 
         public abstract bool CanRead(XmlReader reader);
 
         public abstract void ReadFrom(XmlReader reader);
 
         public override string ToString()
         {
-            return String.Format(CultureInfo.CurrentCulture, "{0}, SyndicationVersion={1}", this.GetType(), this.Version);
+            return string.Format(CultureInfo.CurrentCulture, "{0}, SyndicationVersion={1}", GetType(), Version);
         }
 
         public abstract void WriteTo(XmlWriter writer);
 
         internal protected virtual void SetItem(SyndicationItem item)
         {
-            if (item == null)
-            {
-                throw DiagnosticUtility.ExceptionUtility.ThrowHelperArgumentNull("item");
-            }
-            _item = item;
-        }
-
-        internal static void CreateBufferIfRequiredAndWriteNode(ref XmlBuffer buffer, ref XmlDictionaryWriter extWriter, XmlDictionaryReader reader, int maxExtensionSize)
-        {
-            SyndicationFeedFormatter.CreateBufferIfRequiredAndWriteNode(ref buffer, ref extWriter, reader, maxExtensionSize);
+            Item = item ?? throw new ArgumentNullException(nameof(item));
         }
```

Comment:
This method can be called from a derived class and null passed. Revert null ref exception handling.